seqr/views/apis/auth_api.py

```diff
@@ -28,7 +29,12 @@ def login_view(request):
     if users.count() != 1:
         return create_json_response({}, status=401, reason='Invalid credentials')
 
-    u = authenticate(username=users.first().username, password=request_json['password'])
+    user = users.first()
+    if SOCIAL_AUTH_GOOGLE_OAUTH2_KEY and (user_is_data_manager(user) or user.is_superuser):
```

Comment:
it might be good to make a helper method somewhere for `_is_google_auth_enabled` that you can use both here and in react_app.py

Reply:
Okay